scripts/benchmark/profile_million_allocations

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+#
+# This is intended to be run from within the scripts/benchmark directory of the umpire source.
+# It will take a profile of the develop branch and compare it to a problem of this branch (using hatchet and hpctoolkit)
+#
+
+#
+# Treat undefined variable usage as an error
+#
+set -u
+
+rootdir=`pwd`
+h=`hostname -s | sed -e 's/[0-9]*$//'`
+if [ "$LMOD_arch" = "ppc64le" ]
+then
+   prun="lalloc 1"
+else
+   prun="srun -N1-1 --exclusive"
+fi
+
+run_cmd()
+{
+  echo $1
+  if ! eval $1
+  then
+    echo "FAIL"
+    echo $1
+    echo "FAIL"
+    exit 1
+  fi
+}
+
+# module -q load cmake git gcc/10.2.1 hpctoolkit/2021.05
```

Comment:
Can we make the modules SYS_TYPE dependent or anything?

Reply:
Sure.  Maybe v2?  What we have here will work for TOSS3, TOSS4 (I think), and BLUE systems.

Reply:
Okay, then this looks good, but please remove all the commented lines that look like they were from testing stuff out.